packages/site-server/src/catalog/element-template.ts

```diff
@@ -0,0 +1,95 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import type {Package} from 'custom-elements-manifest/schema.js';
+import {
+  getModule,
+  parseReferenceString,
+  resolveReference,
+  normalizeModulePath,
+} from '@webcomponents/custom-elements-manifest-tools';
+import {escapeHTML} from './escape-html.js';
+
+export const renderElement = ({
+  packageName,
+  elementName,
+  declarationReference,
+  customElementExport,
+  manifest,
+}: {
+  packageName: string;
+  elementName: string;
+  declarationReference: string;
+  customElementExport: string;
+  manifest: Package;
+}): string => {
+  const declarationRef = parseReferenceString(declarationReference);
+  console.log({declarationRef});
+  if (declarationRef.module === undefined) {
+    return `
+      <h1>Error</h1>
+      <p>${declarationReference} has no module</p>
```

Comment:
Should this and the other failures be a 404?

Reply:
Maybe some other 4xx error? In this case the element is found, but some data is malformed. These should be checked in the improt and added to problems. I think making the template either show the errors, or be robust to bad data is the way to do.